README.md

```diff
@@ -1,2 +1,149 @@
-# OverOps
+# OverOps Reliability Report - Teamcity Plugin
 
+This plugin provides a mechanism for applying OverOps severity assignment and regression analysis to new builds to allow application owners, DevOps engineers, and SREs to determine the quality of their code before promoting it into production.
+
+Run this plugin as a post build step after all other testing is complete to generate a Reliability Report that will determine the stability of the build. From the Reliability Report you can drill down into each specific error using the OverOps Automated Root Cause analysis screen to solve the issue.
+
+For more information about the plugin, quality gates, and for regression testing examples, see the Teamcity Plugin Guide.
+
+![OverOps Reliability Report](readme_images/OO_qr.png)
+
+## Installation
+
+   Prerequisites
+
+* Teamcity version
+* Java version 1.8+
+
+Install the OverOps Query Plugins List through the Adminstration Page. From the Teamcity Administration Page, select Plugins List → Browse plugins repository → [JetBrains Plugins](https://plugins.jetbrains.com/teamcity) → scroll down to OverOps Query Plugin → click the get button → select Teamcity server.
```

Comment:
Teamcity -> TeamCity

"scroll down to OverOps Query Plugin" -> "search for OverOps plugin"